tests/acceptance/tests/Account/ForgotPassword.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test } from '@fixtures/AcceptanceTest';
+
+test ('As a customer, I can request a new password with existing customer email address.', { tag: '@Account @Password' }, async ({
+    ShopCustomer,
+    StorefrontAccountLogin,
+    StorefrontAccountRecover,
+    TestDataService,
+}) => {
+    const customer = await TestDataService.createCustomer();
+    await ShopCustomer.goesTo(StorefrontAccountLogin.url());
+    await StorefrontAccountLogin.forgotPasswordLink.click();
+    await StorefrontAccountRecover.emailInput.fill(customer.email);
+    await StorefrontAccountRecover.requestEmailButton.click();
+    await ShopCustomer.expects(StorefrontAccountRecover.passwordResetEmailSentMessage).toBeVisible();
+    await StorefrontAccountRecover.backButton.click();
+    await ShopCustomer.expects(StorefrontAccountLogin.loginButton).toBeVisible();
```

Comment:
To enhance the readability and structure of your test cases, using test.step can be a great approach. It helps to break down the test into logical, understandable steps, making it easier for others to read and maintain. It'd also be great that, we will understand what the test case is doing in detail without understanding the code.

```
await test.step('Navigate to the login page and click on forgot password', async () => {
        await ShopCustomer.goesTo(StorefrontAccountLogin.url());
        await StorefrontAccountLogin.forgotPasswordLink.click();
    });

    await test.step('Fill in the customer email and request a password reset', async () => {
        await StorefrontAccountRecover.emailInput.fill(customer.email);
        await StorefrontAccountRecover.requestEmailButton.click();
    });

    await test.step('Verify the password reset email sent message and navigate back to login', async () => {
        await ShopCustomer.expects(StorefrontAccountRecover.passwordResetEmailSentMessage).toBeVisible();
        await StorefrontAccountRecover.backButton.click();
        await ShopCustomer.expects(StorefrontAccountLogin.loginButton).toBeVisible();
    });
```